otelzap/otelzap_test.go

```diff
@@ -287,13 +290,37 @@ func TestOtelZap(t *testing.T) {
 				requireCodeAttrs(t, m)
 			},
 		},
-	}
+		{
+			log: func(ctx context.Context, log *Logger) {
+				log = log.Clone(WithSetTraceFieldsFunc(func(spanCtx trace.SpanContext) []zapcore.Field {
+					return []zapcore.Field{zap.String("MyTraceId", "123"), zap.String("MySpanID", "456")}
```

Comment:
Should those (I presume) IDs be set from environment variables?

Reply:
No, this is just test data in a test, you can set whatever fields you want to whatever you like. idea being you would use the passed span context. but thats not important for this test.

Reply:
oh zzz 🤦🏾